projects/plugins/jetpack/modules/contact-form/grunion-contact-form.php

```diff
@@ -2264,6 +2275,17 @@ static function parse( $attributes, $content ) {
 
 		$r .= '</div>';
 
+		/**
+		 * Filter the contact form, allowing plugins to modify the HTML.
+		 *
+		 * @module contact-form
+		 *
+		 * @since 10.2
+		 *
+		 * @param string $r The contact form HTML.
+		 */
+		$r = apply_filters( 'jetpack_contact_form_html', $r );
+
 		return $r;
```

Comment:
I think you could potentially put the filter on the return directly:

```suggestion
		return apply_filters( 'jetpack_contact_form_html', $r );
```